package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "alfresco-ng2-components",
   "description": "Alfresco Angular components",
-  "version": "7.0.0-alpha.4",
+  "version": "7.0.0-alpha.5",
```

Comment:
Guys one thing, shouldn't it be `7.0.0-alpha.4`? Last release was `7.0.0-alpha.3` and `7.0.0-alpha.4` is missing

Reply:
This is due to the fact that after release we bump the package version so that new alphas are produced, thats why after releasing alpha3 we bumped it to alpha4